libs/overlay/src/lib/ChatOverlayManager.ts

```diff
@@ -484,6 +494,15 @@ export class ChatOverlayManager {
     return overlay.exportConversation(conversationId);
   }
 
+  public async importConversation(
+    id: string,
```

Comment:
I suggest to rename everywhere for  `id` to `overlayId` to readability into `ChatOverlayManager`
```suggestion
    overlayId: string,
```

Reply:
Fixed